packages/lesswrong/server/emailComponents/NewDialogueMessagesEmail.tsx

```diff
@@ -16,13 +25,32 @@ const NewDialogueMessagesEmail = ({documentId, userId}: {
       version: 'String'
     }
   });
+      
+  const { document: author } = useSingle({
+    documentId: dialogueMessageEmailInfo?.messageAuthorId,
+    collectionName: "Users",
+    fragmentName: "UsersMinimumInfo",
+    skip: !dialogueMessageEmailInfo
+  })
+
   if (!post) return null;
   if (!post.collabEditorDialogue) return null;
   
   const dialogueParticipantIds = [post.userId, ...getConfirmedCoauthorIds(post)];
   
   if (dialogueParticipantIds.includes(userId)) {
     const editUrl = postGetEditUrl(post._id)
+
+    if (dialogueMessageEmailInfo && author) {
+
+      const authorDisplayName = userGetDisplayName(author)
+
+      return (<React.Fragment>
+        <p>{authorDisplayName} left a new reply in your dialogue "<a href={editUrl}>{post.title}</a>".</p>
+        <EmailContentItemBody dangerouslySetInnerHTML={{ __html: dialogueMessageEmailInfo.messageContents }}/> 
+        <p><a href={editUrl}>Click here</a>to respond!</p>
+      </React.Fragment>);
```

Comment:
You don't need to spell out `<React.Fragment>...</React.Fragment>`, can just use `<></>` (which is shorthand for the same)